asmcomp/arm64/emit.mlp

```diff
@@ -1167,7 +1158,7 @@ let begin_assembly() =
   `	.globl	{emit_symbol lbl_begin}\n`;
   `{emit_symbol lbl_begin}:\n`;
   let lbl_begin = Compilenv.make_symbol (Some "code_begin") in
-  emit_named_text_section lbl_begin;
+  emit_named_text_section lbl_begin '%';
```

Comment:
Since the prefix character is always the same for a given backend, I suggest defining a small wrapper around the function in `Emitaux`:
```
let emit_named_text_section lbl =
  Emitaux.emit_named_text_section lbl '%'
```
and use that in the rest of the file.

Reply:
Good idea, done.